data/json/monsters.json

```diff
@@ -122,6 +122,38 @@
     "categories":["WILDLIFE"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_bear_smoky",
+    "name": "Smoky bear",
+    "species":"ZOMBIE",
+    "default_faction":"bear",
+    "symbol":"B",
+    "color":"ltgray",
+    "size":"LARGE",
+    "material":"flesh",
+    "diff":10,
+    "aggression":100,
+    "morale":100,
+    "speed":90,
+    "melee_skill":6,
+    "melee_dice":4,
+    "melee_dice_sides":6,
+    "melee_cut":4,
+    "dodge":3,
+    "armor_bash":4,
+    "armor_cut":4,
+    "vision_day":30,
+    "vision_night":10,
+    "luminance":0,
+    "hp":100,
+    "death_function":"SMOKEBURST",
+    "description":"Smoke bellows from this black-white bear's fur, and the ground bursts into flame in its wake, only to be put out a moment later.",
```

Comment:
Fix this description to reflect that it no longer has fire magic, and this should be ready for merge testing.